src/frontend/js/custom-leaflet.js

```diff
@@ -32,13 +32,13 @@ function printMapAndItinerary() {
     printContainer.remove();
 }
 
-function getWaypoints(routeId) {
+function getRouteWaypoints(routeId) {
     var waypoints = [];
     // Reset current waypoints
     control.setWaypoints(waypoints);
 
     // Ajax call to get waypoint according route
-    $.get( "../getDailyOrders/?route="+routeId, function(data ) {
+    $.get( "../getDailyOrders/?route="+routeId+"&mode=euclidean", function(data ) {
```

Comment:
How to reload a saved route on the map, if we always display the optimized route ?